src/frontend/web_application/src/modules/tags/components/TagsForm/style.scss

```diff
@@ -2,6 +2,8 @@
 @import '../../../../styles/vendor/bootstrap_foundation-sites';
 
 .m-tags-form {
+  width: 100%;
```

Comment:
witdh is required even if you set max-witdh ?

Reply:
yep that's useful for screens narrower than `max-width` (=380px)